lib/matplotlib/axes/_base.py

```diff
@@ -4142,22 +4150,29 @@ def get_tightbbox(self, renderer, call_axes_locator=True,
             self.apply_aspect()
 
         if self.axison:
-            bb_xaxis = self.xaxis.get_tightbbox(renderer)
+            igl = 'x' if for_layout_only else None
+            bb_xaxis = self.xaxis.get_tightbbox(renderer, ignore_label=igl)
```

Comment:
If anyone has written an `Axis` subclass that overrode `Axes.get_tightbox` and did not accept `**kwargs` (which I think is a reasonable thing to do, particularly if they are fully intercepting the call and not forwarding on to `super()`), then this line will fail with a `TypeError` from Python with no warning.

Reply:
Oh. Duh.  Sorry, I'll fix...

Reply:
Did (for both axis-es):

```python
            try:
                bb_xaxis = self.xaxis.get_tightbbox(renderer, ignore_label=igl)
            except TypeError as e:
                # in case downstream library has redefined axis:
                bb_xaxis = self.xaxis.get_tightbbox(renderer)
```